examples/StandardFirmataWiFi/StandardFirmataWiFi.ino

```diff
@@ -309,6 +309,12 @@ void checkDigitalInputs(void)
 }
 
 // -----------------------------------------------------------------------------
+// function forward declarations
+void enableI2CPins();
```

Comment:
Strange I have not run into any compiler errors when using the ESP8266 Arduino core and I have successfully compiled each module that library supports (except ThaiEasyElec's ESPino which fails). At least not as of version 2.1.0.